service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/EmailSenderProvider.java

```diff
@@ -4,24 +4,14 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.mail.javamail.JavaMailSender;
 import org.springframework.stereotype.Service;
-import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
 
 @Service
 public class EmailSenderProvider {
```

Comment:
tbh, i would remove this class now completely and inject JavaMailSender into EmailService.

aditionally i would remove     @Bean("sendGridMailSender") from SendGridMailSenderConfiguration